### DISCUSSION TITLE
yview for ScrollableFrame

### DISCUSSION THREAD

**Discussion by UserJoo9** [TomSchimansky/CustomTkinter#2833] [Q&A]
if any way to scroll yview in ScrollableFrame automatically??

**Comment by MaxNiark** [comment]
Hello, 

Many example in the network .. 

If you find a way for scroll X AND Y  in the same frame, and on toplevel. please trade that <3

**Comment by UserJoo9** [comment]
@MaxNiark 
i found only one way to do this
by using this line

`<your_frame_object>._parent_canvas.yview("scroll", int(2500 / 6), "units")`

use it by replacing <your_frame_object> with your ScrollableFrame object name
it will scroll down in any reason
and you can use this line 

`<your_frame_object>._parent_canvas.yview("scroll", -int(2500 / 6), "units")`

to scroll up in any reason also

**Comment by MaxNiark** [comment]
Hey, please find setup in _init_ CTKscrolableframe 


![image](https://user-images.githubusercontent.com/126392328/233976653-b5790843-a191-4f38-a499-35988f2f8ba5.png)
![image](https://user-images.githubusercontent.com/126392328/233976762-a9d15621-b055-48d3-82d5-dfbe9d0af648.png)

**Comment by MaxNiark** [comment]
holy automatically ? 

what do you mean ? 

When you pressure a keyboard or mouse ?

**Comment by MaxNiark** [comment]
Binding of scrollable frame 

![image](https://user-images.githubusercontent.com/126392328/233977428-0385199e-2b80-4cbb-a34f-d50ab1c573dc.png)

**Comment by MaxNiark** [comment]
if you would like yscroll the screen unit, it's needed to create a fonction for _parent_canvas.

<ITEM>.move() 
<Item>.after(ms,function)  <= Becarefull this is not run as multithreading ... 1Ms is soo annoying ifyou get a big app

something like this : 
![image](https://user-images.githubusercontent.com/126392328/233978198-5fde8cb0-e486-485f-bcde-ac52c2e259b2.png)

![image](https://user-images.githubusercontent.com/126392328/233978038-0e9479cd-1ad1-46c6-981c-c96ffd5f246e.png)
